src/content/docs/en/guides/integrations-guide/vue.mdx

```diff
@@ -194,3 +194,16 @@ export default defineConfig({
   ],
 });
 ```
+
+### devtools
+
+You can enable [`vuejs/devtools-next`](https://github.com/vuejs/devtools-next) in development by setting `devtools: true`.
```

Comment:
```suggestion
You can enable [Vue DevTools](https://devtools-next.vuejs.org/) in development by setting `devtools: true`.
```
What do you think about linking to the docs/marketing site? Going to GitHub directly doesn't tell you very much, and its own README just points here for even installation and setup.